### PR TITLE
feature: pipelineyml parser add missing template funcs

### PR DESCRIPTION
**Pull request by sfwn**
#### What type of PR is this?

feature

#### What this PR does / why we need it:

pipelineyml parser add missing template funcs, other modules will use it.

#### Which issue(s) this PR fixes:

Fixes #

none

### CODE REVIEW

File: pkg/parser/pipelineyml/visitor_snippet.go [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/pkg/expression"
 )
 
 const (
 	Snippet            = "snippet"
 	SnippetLogo        = "http://terminus-paas.oss-cn-hangzhou.aliyuncs.com/paas-doc/2020/10/22/410935c6-e399-463a-b87b-0b774240d12e.png"
 	SnippetDisplayName = "嵌套流水线"
 	SnippetDesc        = "嵌套流水线可以声明嵌套的其他 pipeline.yml"
+
```
Comment (liuhaoyang) [inline pkg/parser/pipelineyml/visitor_snippet.go:21]: `SnippetLogo` address needs to be obtained using configuration

**Review by Effet — APPROVED** [review]
i know it's legacy code for compatibility, pls refactor it later bro.

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: pkg/parser/pipelineyml/visitor_snippet.go (hunk 1 above)
Comment (sfwn) [reply pkg/parser/pipelineyml/visitor_snippet.go:21]: no need now

Comment (sfwn) [reply pkg/parser/pipelineyml/visitor_snippet.go:21]: this file will be delete totally soon. after all modules migrate to erda, then adjust it.

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**